lib/matplotlib/animation.py

```diff
@@ -744,14 +747,15 @@ class ImageMagickFileWriter(ImageMagickBase, FileMovieWriter):
 
     Frames are written to temporary files on disk and then stitched
     together at the end.
-
     """
 
-    supported_formats = ['png', 'jpeg', 'ppm', 'tiff', 'sgi', 'bmp',
-                         'pbm', 'raw', 'rgba']
+    supported_formats = ['png', 'jpeg', 'ppm', 'tiff', 'sgi', 'bmp', 'pbm',
+                         'rgba']
```

Comment:
Isn't 'raw' the same as 'rgba'?

Reply:
not from the PoV of ImageMagick, but I changed that PR to make instead force ImageMagick to interpret the files as rgba.